packages/deployments/contracts/contracts/lib/Connext/ConnextLogic.sol

```diff
@@ -71,6 +75,24 @@ library ConnextLogic {
     address bridged;
   }
 
+  struct ReconcileArgs {
+    uint32 origin;
+    bytes message;
+    ITokenRegistry tokenRegistry;
+    address aavePool;
+    uint256 portalFeeNumerator;
+    uint256 portalFeeDenominator;
+  }
+
+  struct ReconcilePortalVars {
+    uint256 portalTransferAmount;
+    uint256 adoptedAmountOut;
+    address adopted;
+    uint256 totalRepayAmount;
+    uint256 backUnbackedAmount;
+    uint256 portalFee;
```

Comment:
isn't this set by aave? is there no way to pull it from there instead of specifying it in the args?